deploy/olm-catalog/meta-operator/0.0.1/meta-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -87,9 +87,10 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    categories: Cloud App Manager
+    categories: Developer Tools, Monitoring, Logging & Tracing, Security
```

Comment:
Why change cloud app manager to this?

Reply:
To pass the operator-courier verify.
I check the valid category list: https://github.com/operator-framework/operator-courier/blob/master/operatorcourier/validate.py#L579

Otherwise, it will report an error: 
```
ERROR: category Cloud App Manager is not a valid category [meta-operator/meta-operator.package.yaml]
```